### PR TITLE
enhancement of docker-extensions UI

### PR DESCRIPTION
**Pull request by sudhanshutech**
#### 1. Enhance the walkthrough 
image

#### 2. Added docs button to redirect to meshery docs
image

#### 3. Added view all catalog redirect to cloud catalog
image

#### 5. Add feedback component
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js [hunk 1]
```diff
@@ -132,6 +133,36 @@ const ExtensionsComponent = () => {
...
+    //   return;
+    // }
+    // const path = router.pathname;
+    const userFeedbackRequestBody = {
+      scope: feedback?.label,
+      message: feedback?.message,
+      page_location: "",
+      metadata: {}
+    };
+    fetch(proxyUrl + '/api/extensions/identity/users/notify/feedback', {
```
Comment (sudhanshutech) [inline install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js:150]: The api giving error of cannot `Error: Status Code: 405.failed to request to remote provider`.
Am i missing something? // @humblenginr @MUzairS15

Comment (humblenginr) [reply install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js:150]: do not use `fetch` directly. Use `window.ddClient`.

**Comment by leecalcote** [comment]
@sudhanshutech, what response do you get when attempting to invoke the feedback API?

File: install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js [hunk 2]
```diff
@@ -558,6 +615,21 @@ const ExtensionsComponent = () => {
             </div>
           )}
         </SectionWrapper>
+
+
+        {/* 
+        
+        // Feedback component is comment currently because the api required to use this authentication error
```
Comment (sudhanshutech) [inline install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js:622]: Currently i have commented out the feedback component though the functionality is there until we able to use the api. II tried to set the cookie in header but it didn't solved. // @MUzairS15 @humblenginr

**Comment by sudhanshutech** [comment]
Its good to go.
I have added now :
1. A docs which will redirect user to meshery docs
2. A view all catalog so user can see all cloud catalog
3. And catalog card is also redirect to single cloud catalog page

**Pull request merged**